### PR TITLE
new model, create_game funtion and tests directory

### PR DESCRIPTION
**Pull request by Gerson2102**

### CODE REVIEW

**Comment by Gerson2102** [comment]
I still have some errors left.

File: src/models/board.cairo [hunk 1]
```diff
@@ -1,4 +1,3 @@
-#[derive(Model, Copy, Drop, Serde)]
```
Comment (Jonatan-Chaverri) [inline src/models/board.cairo:1]: Don't remove this, we need it to store it in the blockchain

File: src/models/graph_node.cairo [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#[derive(Model, Copy, Drop, Serde)]
+struct GraphNode {
```
Comment (Jonatan-Chaverri) [inline src/models/graph_node.cairo:2]: This do not exists anymore

File: src/tests/create_game.cairo [hunk 3]
```diff
@@ -0,0 +1,6 @@
+use starktrip::store::{Store, StoreTrait};
+
+
+#[test]
+#[available_gas(1_000_000_000)]
+fn create_game() {}
```
Comment (Jonatan-Chaverri) [inline src/tests/create_game.cairo:6]: Remove this empty test

**Review by Jonatan-Chaverri — APPROVED** [review]

**Pull request merged**